pkg/dss/auth/scopes.go

```diff
@@ -0,0 +1,34 @@
+package auth
+
+// Operation models the name of an operation.
+type Operation string
```

Comment:
Let's document the formatting required here -- maybe something like "grpc.UnaryServerInfo.FullMethod of operation, generally formatted like `/{package-qualified service name}/{handler name}`"